task/jefe/build.rs

```diff
@@ -142,5 +148,30 @@ fn output_dump_areas(out: &mut std::fs::File) -> Result<()> {
 
     writeln!(out, "];")?;
 
+    writeln!(
+        out,
+        r##"
+///
+/// Function to determine if an address/length pair is contained within a
+/// dump area, short-circuiting in the common case that it isn't.  (Note that
+/// this will only check for containment, not overlap.)
+///
+pub(crate) fn in_dump_area(address: u32, length: u32) -> bool {{
```

Comment:
Nit: does this actually need to be generated?  It looks like it could go into `jefe/src/dump.rs` without any issues.

Reply:
I auto-generated it just to allow the short-circuiting min/max check to be done. Clarified this in the comment.